`azurerm_monitor_action_group` `webhook_receiver.0.aad_auth.0.identifier_uri` expects "api" scheme

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

<!--- Please keep this note for the community --->

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

<!--- Thank you for keeping this note for the community --->


### Terraform Version

1.5.5

### AzureRM Provider Version

3.68.0

### Affected Resource(s)/Data Source(s)

azurerm_monitor_action_group

### Terraform Configuration Files

```hcl
resource "azurerm_monitor_action_group" "this" {
  resource_group_name = "rg"
  name                = "test"
  short_name          = "test"
  enabled             = true

  webhook_receiver {
      name                    = "webhook"
      use_common_alert_schema = true
      service_uri             = "https://webhook.site.io/azure-alerts"
      aad_auth {
        object_id      = "REDACTED"
        tenant_id      = "REDACTED"
        identifier_uri = "https://webhook.site.io"
      }
    }
  }
}
```


### Debug Output/Panic Output

```shell
2023-08-09T22:47:55.679-0400 [ERROR] provider.terraform-provider-azurerm_v3.68.0_x5.exe: Response contains error diagnostic: diagnostic_attribute=AttributeName("webhook_receiver").ElementKeyInt(0).AttributeName("aad_auth").ElementKeyInt(0).AttributeName("identifier_uri") diagnostic_summary="expected "webhook_receiver.0.aad_auth.0.identifier_uri" to have a url with schema of: "api", got https://webhook.site.io" tf_proto_version=5.3 tf_resource_type=azurerm_monitor_action_group @module=sdk.proto diagnostic_detail= tf_req_id=89bcffcd-6ca1-a501-9a02-89f70885ee64 tf_rpc=ValidateResourceTypeConfig diagnostic_severity=ERROR tf_provider_addr=provider @caller=github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/diag/diagnostics.go:55 timestamp=2023-08-09T22:47:55.679-0400
```


### Expected Behaviour

The identifier uri should be allowed as it is valid for our AAD tenant.

### Actual Behaviour

Error: expected "webhook_receiver.0.aad_auth.0.identifier_uri" to have a url with schema of: "api", got https://webhooks.site.io

### Steps to Reproduce

terraform plan

### Important Factoids

_No response_

### References

_No response_

--- a/internal/services/monitor/monitor_action_group_resource.go
+++ b/internal/services/monitor/monitor_action_group_resource.go
@@ -228,7 +228,7 @@ func resourceMonitorActionGroup() *pluginsdk.Resource {
 										Type:         pluginsdk.TypeString,
 										Optional:     true,
 										Computed:     true,
-										ValidateFunc: validation.IsURLWithScheme([]string{"api"}),
+										ValidateFunc: validation.IsURLWithScheme([]string{"api", "https"}),
 									},
 
 									"tenant_id": {

